js/ko_chance.js

```diff
@@ -158,55 +155,76 @@
 		qualifier = 'approx. ';
 		damage = squashMultihit(damage, hits);
 	}
-
-	var c = getKOChance(damage, defender.maxHP - hazards, 0, 1, defender.maxHP, toxicCounter);
-	var afterText = hazardText.length > 0 ? ' after ' + serializeText(hazardText) : '';
-	if (c === 1) {
-		return 'guaranteed OHKO' + afterText;
-	} else if (c > 0) {
-		return qualifier + Math.round(c * 1000) / 10 + '% chance to OHKO' + afterText;
-	}
-
-	afterText = hazardText.length > 0 || eotText.length > 0 ? ' after ' + serializeText(hazardText.concat(eotText)) : '';
-	var i;
-	for (i = 2; i <= 4; i++) {
-		c = getKOChance(damage, defender.maxHP - hazards, eot, i, defender.maxHP, toxicCounter);
+	var c;
+	var afterText = hazardText.length > 0 || eotText.length > 0 ? ' after ' + serializeText(hazardText.concat(eotText)) : '';
+	if (move.usedTimes === 1) {
+		c = getKOChance(damage, defender.maxHP - hazards, 0, 1, defender.maxHP, toxicCounter);
+		afterText = hazardText.length > 0 ? ' after ' + serializeText(hazardText) : '';
 		if (c === 1) {
-			return 'guaranteed ' + i + 'HKO' + afterText;
+			return 'guaranteed OHKO' + afterText;
 		} else if (c > 0) {
-			return qualifier + Math.round(c * 1000) / 10 + '% chance to ' + i + 'HKO' + afterText;
+			return qualifier + Math.round(c * 1000) / 10 + '% chance to OHKO' + afterText;
+		}
+		var i;
+
+		for (i = 2; i <= 4; i++) {
+			c = getKOChance(damage, defender.maxHP - hazards, eot, i, 1, defender.maxHP, toxicCounter);
+			if (c === 1) {
+				return 'guaranteed ' + i + 'HKO' + afterText;
+			} else if (c > 0) {
+				return qualifier + Math.round(c * 1000) / 10 + '% chance to ' + i + 'HKO' + afterText;
+			}
+		}
+
+		for (i = 5; i <= 9; i++) {
+			if (predictTotal(damage[0], eot, i, 1, toxicCounter, defender.maxHP) >= defender.maxHP - hazards) {
+				return 'guaranteed ' + i + 'HKO' + afterText;
+			} else if (predictTotal(damage[damage.length - 1], eot, i, 1, toxicCounter, defender.maxHP) >= defender.maxHP - hazards) {
+				return 'possible ' + i + 'HKO' + afterText;
+			}
 		}
-	}
 
-	for (i = 5; i <= 9; i++) {
-		if (predictTotal(damage[0], eot, i, toxicCounter, defender.maxHP) >= defender.maxHP - hazards) {
-			return 'guaranteed ' + i + 'HKO' + afterText;
-		} else if (predictTotal(damage[damage.length - 1], eot, i, toxicCounter, defender.maxHP) >= defender.maxHP - hazards) {
-			return 'possible ' + i + 'HKO' + afterText;
+	} else {
+		qualifier = 'nearly ';
+		// until someone comes up with a better damage recalculation formula, we'll tell the user it's an estimate
```

Comment:
I don’t know if this is necessary..

Reply:
As mentioned, the multi-turn moves formula is a bit brutal, but couldn't find a more accurate way to deal with the new attack value.